sdk-api/src/main/java/com/manywho/sdk/api/run/elements/config/ServiceRequest.java

```diff
@@ -70,4 +70,12 @@ public FlowMode getExecutionMode() {
     public void setExecutionMode(FlowMode executionMode) {
         this.executionMode = executionMode;
     }
+
+    public boolean isDebugMode() {
+        if (executionMode == null) {
+            return false;
+        }
+
+        return executionMode.equals(FlowMode.Debug) || executionMode.equals(FlowMode.DebugStepthrough);
```

Comment:
I think the problem here is that executionMode is declared in the parent and also in this class. Could you try to remove the declaration of this variables in ServiceRequest, and also remove this method add check if that fix the problem?

Reply:
I removed duplicated/redefined vars and methods from ServiceRequest.